src/Docfx.Build/HostServiceCreator.cs

```diff
@@ -63,8 +63,12 @@ public HostServiceCreator(DocumentBuildContext context)
             {
                 if (e is not DocumentException)
                 {
+                    var message = e is ArgumentException or NullReferenceException
+                        ? e.ToString()
+                        : e.Message;
+
                     Logger.LogError(
-                        $"Unable to load file '{file.File}' via processor '{processor.Name}': {e.Message}",
+                        $"Unable to load file '{file.File}' via processor '{processor.Name}': {message}",
```

Comment:
Ah, we should have a `LogError` overload that takes an `Exception` that logs exception message as error and full exception as verbose.